mmdet/apis/test.py

```diff
@@ -14,6 +15,69 @@
 from mmdet.core import encode_mask_results
 
 
+def ipu_test(model, data_loader, show=False, out_dir=None, show_score_thr=0.3):
+    model.eval()
+    results = []
+    dataset = data_loader.dataset
+    PALETTE = getattr(dataset, 'PALETTE', None)
+    prog_bar = mmcv.ProgressBar(len(dataset))
+    for i, data in enumerate(data_loader):
+        data['img_metas'] = data['img_metas'][0].data
+        with torch.no_grad():
+            result = model(return_loss=False, rescale=True, **data)
+            result = model.model.model.post_process(result)
```

Comment:
model.model.model?

Reply:
The model needs to be warp to compile, so the real model is model.model.model

Reply:
looks a little weird.

Reply:
okay I will fix it